actors/verifreg/src/lib.rs

```diff
@@ -675,6 +684,167 @@ impl Actor {
             data_cap_removed: removed_data_cap_amount,
         })
     }
+
+    pub fn revoke_expired_allocations<BS, RT>(
```

Comment:
Could you add a brief documentation comment describing the behaviour here? Mostly copy from the FIP.